src/i18n/translation-checkers.ts

```diff
@@ -27,7 +27,7 @@ export const NavDictionary = (dict: Partial<Record<NavDictionaryKeys, string>>)
 	const orderedDictionary: NavDict = [];
 	for (const enEntry of enNav) {
 		const text = dict[enEntry.key] || enEntry.text;
-		orderedDictionary.push({ ...enEntry, text });
+		orderedDictionary.push({ ...enEntry, text, isFallback: !dict[enEntry.key] });
```

Comment:
Why was this necessary exactly? Usually our `isFallback` flag does not flag that the _label_ is a fallback, but that the page contents itself are (we use it to display an “EN” suffix in the sidebar for such pages even if the label is translated).

Reply:
Ahh, that explains things. Here, I used this to detect if the label was falling back to english, to check if all labels were provided before marking it as "complete". I'll move this out to a separate value tomorrow